ci-operator/step-registry/openshift/e2e/ovirt/clusterbot/openshift-e2e-ovirt-clusterbot-workflow.yaml

```diff
@@ -0,0 +1,24 @@
+workflow:
+  as: openshift-e2e-ovirt-clusterbot
+  steps:
+    pre:
+    - chain: ipi-ovirt-pre
+    - ref: ipi-conf-ovirt-generate-csi-test-manifest
+    - ref: storage-obj-save
+    test:
+    - as: e2e-test
+      commands: sleep $(CLUSTER_DURATION) & wait
+      from_image:
+        namespace: origin
+        name: centos
+        tag: '8'
+      resources:
+        requests:
+          cpu: 100m
+          memory: 200Mi
+    post:
+    - ref: storage-obj-check
+    - chain: ipi-ovirt-post
+  documentation: |-
+    The Openshift E2E oVirt `csi` workflow installs OpenShift cluster on top of oVirt
+    Executes the `openshift/csi` end-to-end test suite. This is a copy of openshift-upgrade-ovirt workflow using a different lease and custom test
```

Comment:
The Openshift E2E oVirt `clusterbot` workflow installs an OpenShift cluster on top of oVirt and for testing and development purposes via the slack clusterbot, see https://github.com/openshift/ci-chat-bot